cruz_solo_assessment/db/migrate/20150527133939_create_people.rb

```diff
@@ -0,0 +1,4 @@
+class CreatePeople < ActiveRecord::Migration
+  def change
```

Comment:
You use Migration objects to create new tables, not to pull resources out of your database